### PR TITLE
Responsiveness added

### PR DESCRIPTION
**Pull request by JGDel24**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,4 +1,200 @@
...
+
+#Connect ul {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+}
+
+
+/*Tablet style*/
+@media only screen and (min-width: 600px) and (min-width: 900px) {
```
Comment (brandaogabriel7) [inline css/index.css]: You're using two different values for `min-width`, that condition will never be fulfilled. I think you meant to use `max-width` for the second one.

```suggestion
@media only screen and (min-width: 600px) and (max-width: 900px) {
```

File: css/index.css
Comment (brandaogabriel7) [inline css/index.css:1]: Don't forget to format your file to ensure consistent indentation. There are sections that are missing some indent levels.

**Review by brandaogabriel7 — CHANGES_REQUESTED** [review]
Nice job implementing responsiveness, Joel!

You just need to fix the issue with using `min-width` instead of `max-width` and it's good to go!

File: css/index.css (hunk 1 above)
Comment (JGDel24) [reply css/index.css]: thank you! I will get right on this!

File: css/index.css
Comment (JGDel24) [reply css/index.css:1]: Thank You I will update this

**Pull request merged**